cdap-api/src/main/java/co/cask/cdap/api/annotation/Batch.java

```diff
@@ -70,7 +70,7 @@
  * </p>
  * 
  * <p>
- * See the <i><a href="http://cask.co/docs/cdap/current/en/">Cask DAP Developer Guides</a></i>
+ * See the <i><a href="http://cask.co/docs/cdap/current/en/developers-manual/index.html">CDAP Developers' Manual</a></i>
```

Comment:
should it be Developer's? Also, that http://cask.co/docs/cdap/current/en/developers-manual/index.html is a 404